pkg/node/getdatastoreentries.go

```diff
@@ -31,17 +36,38 @@ func (u JSONableSlice) MarshalJSON() ([]byte, error) {
 }
 
 func DatastoreEntry(client *Client, address string, key string) (*DatastoreEntryResponse, error) {
+	entries := []GetDatastoreEntriesString{}
+	entry := GetDatastoreEntriesString{
+		Address: address,
+		Key:     key,
+	}
+	entries = append(entries, entry)
+
+	response, err := DatastoreEntries(client, entries)
+	if err != nil {
+		return nil, err
+	}
+	result := *(response)
+	return &result[0], nil
+}
+
+func DatastoreEntries(client *Client, params []GetDatastoreEntriesString) (*[]DatastoreEntryResponse, error) {
```

Comment:
In go, arrays are passed as slices. And slices are references. So you don't need to return a pointer here, the array itself is sufficiently efficient.

See https://go.dev/doc/effective_go#slices